docs/user_guide/Initial_Server_Configuration.rst

```diff
@@ -112,29 +129,27 @@ method.
         If this happens, you can debug by either looking at the log files or by
         running ``puppet agent -t --masterport=8150``.
 
-#. Type ``reboot`` to reboot and apply the necessary kernel configuration items.
+#. Run ``reboot`` to restart your system and apply the necessary kernel
+   configuration items.
 
-Optional: Extract the full OS Package Set
------------------------------------------
+Optional: Extract the full OS RPM Package Set
+---------------------------------------------
 
-The SIMP ISO attempts to contain everything that you need to run a base system.
-However, if you did not install via ISO, or you require additional stock
-packages, you can use the following procedure to extract the vendor ISOs.
+The SIMP ISO only provides enough RPM packages to run a basic system. If you
+did not install via ISO, or you require additional stock packages, you can
+extract additional packages from vendor ISOs using the following procedure:
 
 #. Log on as ``simp`` and run ``su -`` to gain root access.
-#. Run puppet for the first time.
-
-   Type: ``puppet agent -t``
-
+#. Run ``puppet agent -t`` to ensure system consistency.
 #. Copy the appropriate vendor OS ISO(s) to the server and unpack using the
-   ``unpack_dvd`` utility. This creates a new tree under
+   ``unpack_dvd`` utility. This will create a new directory tree under
    ``/var/www/yum/<OperatingSystem>`` suitable for serving to clients.
 
-   Type: ``unpack_dvd CentOS-RHEL_MAJOR_VERSION-x86_64-DVD-####.iso``
+   Run: ``unpack_dvd CentOS-RHEL_MAJOR_VERSION-x86_64-DVD-####.iso``
 
 #. Update your system using :term:`yum`. The updates applied will depend on
    what ISO you initially used.
 
-   Type: ``yum clean all; yum makecache``
+   Run: ``yum clean all; yum makecache``
```

Comment:
This updates the yum repo, but doesn't update the server.  Should we include `yum update` after this ?

Reply:
Puppet will take care of any necessary updates. Don't see the need to do this by hand.